README.md

```diff
@@ -37,17 +37,20 @@ Currently, chDB only supports `query` function, which is used to execute SQL and
 
 ```python
 import chdb
-res = chdb.query('select version()', 'CSV'); print(str(res.get_memview().tobytes()))
+res = chdb.query('select version()', 'JSON'); print(str(res.get_memview().tobytes()))
```

Comment:
Not sure if JSON is a good default example from a performance and response size / readability point of view.

Reply:
I think JSON is okay, as it's so widely used

Reply:
Then perhaps JSONCompact is a good compromise. This is what play UI uses as well.